components/game/CardDetail.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+
+import { rarityMap, rarityColorKey, toSentenceCase } from '../../lib/utils'
+
+import { ICard, CardType } from '../../lib/game/types'
+
+interface CardDetailProps {
+  card: ICard
+}
+
+const CardDetail = ({ card }: CardDetailProps) => {
+  return (
+    <div
+      className={`flex flex-col justify-between w-72 h-96 p-4 bg-background
+          rounded-md shadow-xl border-4 border-${rarityColorKey(card.level)} `}
+    >
+      <div className="flex flex-col space-y-3 overflow-y-scroll no-scrollbar">
+        <p className={``}>{card.name}</p>
+
+        <p className={`text-${rarityColorKey(card.level)}`}>
+          {rarityMap[card.level]} {toSentenceCase(card.type)}
+        </p>
+
+        {card.type === CardType.SPELL &&
+          card.effects.map((effect, idx) => (
+            <div key={idx} className="flex flex-col">
+              <div className="flex flex-row justify-between items-center pb-2">
+                <p className="py-2">Deal 1 damage to your opponent</p>
```

Comment:
Feels like we should extract this string from this view. 

Don't need to overengineer this, but thinking we should find a better home – whether a strings library (to aid w/ eventual i18n internationalization) or place into a quick and dirty game/constants file.

Reply:
ahh. looking a few lines down, looks like do this already for `{ability.name}` and might be more of a lack of spell development rn.

Reply:
Yea - this is a dirty hack. In the future this string should probably come from something like 'effect.description'. Good call out on i18n - I've neglected that so far.

Reply:
no worries! fine to take on some technical debt to 🚢 faster as long as we're mindful of it.

rn we have only `[🚀 Feature Request,  🐛 Bug Report]` per our [Create an issue](https://github.com/Adventure-Cards/deck-viewer/issues/new/choose) page, but in the near future I'd like to create some templates for  `[🎨 Enhancement,  🏗️ Code Cleanup, 🗺️ i18n ]` to help spur some additional contributions or enable us to punt work like this (e.g. non-urgent,  non-glamorous)  to our backlog.